src/faldbt/parse.py

```diff
@@ -103,6 +103,25 @@ def get_dbt_manifest(config) -> Manifest:
     return ManifestLoader.get_full_manifest(config)
 
 
+def get_dbt_sources_artifact(project_dir: str, config: RuntimeConfig):
+    sources_path = os.path.join(project_dir, config.target_path, "sources.json")
+    try:
+        # BACKWARDS: Change intorduced in 1.0.0
+        if hasattr(FreshnessExecutionResultArtifact, "read_and_check_versions"):
+            return FreshnessExecutionResultArtifact.read_and_check_versions(
+                sources_path
+            )
+        else:
+            return FreshnessExecutionResultArtifact.read(sources_path)
+
+    except IncompatibleSchemaException as exc:
+        exc.add_filename(sources_path)
+        raise
+    except RuntimeException as exc:
```

Comment:
In what cases does this happen?

Reply:
when the `target/sources.json` file is not there